src/openforms/config/models/config.py

```diff
@@ -558,6 +559,16 @@ class GlobalConfiguration(SingletonModel):
         ),
     )
 
+    referentielijsten_services = models.ManyToManyField(
+        Service,
+        verbose_name=_("referentielijsten services"),
+        help_text=_(
+            "List of services that are instances of the Referentielijsten API. "
+            "The selected services will be shown as options when configuring "
+            "select or selectboxes components to be populated from Referentielijsten."
+        ),
+    )
```

Comment:
The test failures seem to be from the admin tests for global configuration - perhaps it's because this field is not marked as `blank=True` which makes it required now, while it shouldn't be?

Reply:
changed it :+1: